user-interface/src/data-verification/ConsolidationOrderAccordion.tsx

```diff
@@ -594,11 +679,11 @@ export function ConsolidationOrderAccordion(props: ConsolidationOrderAccordionPr
                       <CamsSelect
                         id={'lead-case-court'}
                         required={true}
-                        options={getOfficeList(filteredOfficesList ?? props.officesList)}
+                        options={leadCaseSelectOptions}
                         onChange={handleSelectLeadCaseCourt}
                         ref={leadCaseDivisionRef}
                         label="Select a court"
-                        value={getUniqueDivisionCodeOrUndefined(order.childCases)}
+                        value={leadCaseSelectValue}
```

Comment:
For a reason I have now forgotten, this was getting us into an infinite loop.  To simplify things, I moved this code into 2 functions and used state variables to update the component.